suite/tests/runmulti.cmake

```diff
@@ -102,9 +103,9 @@ macro(process_cmdline line skip_empty err_and_out)
         RESULT_VARIABLE cmd_result
         ERROR_VARIABLE cmd_err
         OUTPUT_VARIABLE cmd_out)
-      if (cmd_result)
+      if (cmd_result AND NOT failok)
```

Comment:
We needed this because otherwise `phys_nopriv_failok` would fail in the tracing part, and wouldn't even run the view tool? Isn't that okay?

Reply:
It would fail the test outright and we would have no point to verify the message about failing because of privileges and then *pass* the test.  These tests pass or fail based on the exit code of runmulti.cmake.